src/frontend/src/lib/components/dapps/DappsCarouselSlide.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import { type CarouselSlideOisyDappDescription } from '$lib/types/oisyDappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dappsCarouselSlide: CarouselSlideOisyDappDescription;
+	$: ({
+		carousel: { text, callToAction },
+		logo,
+		name
+	} = dappsCarouselSlide);
+</script>
+
+<div class="flex">
+	<div class="mr-4 shrink-0">
+		<Img
+			height="64"
+			width="64"
+			rounded
+			src={logo}
+			alt={replacePlaceholders($i18n.dapps.alt.logo, { $dAppname: name })}
+		/>
+	</div>
+	<div>
+		<div class="mb-1">{text}</div>
+		<button
```

Comment:
Can you use the `Button` component here? Or it does not fits the needs?

Reply:
I just tried it - it comes with too many things and apparently I cannot turn it off via props. So, if it's okay with you, I'd just use `button` here

Reply:
yes, makes sense